src/utils/gameBoardHelper.ts

```diff
@@ -0,0 +1,157 @@
+import { Letter, PlayerIndex, SparseArray } from "../types";
+
+export function verticalSlice<T>(
```

Comment:
It's actually a column of `Letter`s as in all the letters from a column from the game board. Just named it "verticalSlice" because `Play` uses horizontal/vertical.